syntax = "proto3";
option cc_generic_services = true;

message RPCRequest {
    string client_name = 1;
}
message RPCResponse {
    string serial_number = 1;
}
service DemoService {
    rpc GetSerialNumber(RPCRequest) returns (RPCResponse);
    rpc GetAnotherSerialNumber(RPCRequest) returns (RPCResponse);
}